src/core/service.tf

```
@@ -0,0 +1,40 @@
+resource "azurerm_monitor_scheduled_query_rules_alert" "service_availability_manageservices" {
+  name                = format("[%s] %s %s", upper("service"), "availability", "manageservices")
+  resource_group_name = azurerm_resource_group.rg_external.name
+  location            = azurerm_resource_group.rg_external.location
+
+  action {
+    action_group = [azurerm_monitor_action_group.quarantine_error_action_group]
+  }
+
+  data_source_id          = var.env_short == "p" ? data.azurerm_key_vault_secret.sec_workspace_id[0].value : data.azurerm_log_analytics_workspace.monitor_rg.id
+  description             = "Availability for get and manage services is less than or equal to 99%, check ${module.app_gw.name}."
```

Not necessary, the alert page will provide names and links we need.

Maybe we can think about using a variable for defining the threshold.

```suggestion
  description             = "Availability for get and manage services is less than or equal to 99%."
```